src/CitizenAction/CitizenActionCommandHandler.php

```diff
@@ -47,13 +46,4 @@ public function handleUpdate(CitizenActionCommand $command, CitizenAction $actio
 
         return $action;
     }
-
```

Comment:
it moved in `EventCancelHandler` to handle the cancelation of Committee Event or of Citizen Action in the same class